api/controllers/todoListController.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+const Todo = mongoose.model('Todos')
+
+exports.list_all_todos = (req, res) => {
+  Todo.find({}, (err, todos) => {
+    if(err) {
```

Comment:
on line 16, you have a space after `if` but before `(`, and here you don't. should be consistent